exonum-java-binding-core/rust/src/proxy/node.rs

```diff
@@ -65,8 +63,8 @@ impl NodeContext {
     }
 
     #[doc(hidden)]
-    pub fn submit(&self, transaction: Box<Transaction>) -> io::Result<()> {
-        self.channel.send(transaction)
+    pub fn submit(&self, transaction: RawTransaction) -> Result<(), failure::Error> {
+        self.blockchain.broadcast_raw_transaction(transaction)
```

Comment:
This name is a little misleading, as broadcast does not imply any extra processing (signing). I'd add a comment here, or improve the naming (and add docs) in the core.

Reply:
This is technically our internal API, I think it's okay know as we will change this mechanism in 1.0 anyway

Reply:
However, I will consider changes in core